### ISSUE TITLE
[5.3.x] 'bazel info' reports 'unknown key(s)' when specifying '--no//flag'

### ISSUE THREAD

**Issue by beasleyr-vmw** [bazelbuild/bazel#16257]
### Description of the bug:

Looks like this was already fixed by a Googler at f6cccae5b6f9c0ad0e7d0bf7bd31ea1263449316 and would be a trivial backport to 5.3.x.

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

(NOTE: It doesn't matter if WORKSPACE contains a repository rule for bazel_skylib.)

```console
$ cd $(mktemp -d)
$ touch WORKSPACE
$ cat >BUILD <<EOF
load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")

bool_flag(
    name = "example",
    build_setting_default = True,
)
EOF
$ USE_BAZEL_VERSION=5.3.0 bazelisk info --no//:example
ERROR: unknown key(s): '--no//:example'
```

### Which operating system are you running Bazel on?

Linux

### What is the output of `bazel info release`?

release 5.3.0

### Have you found anything relevant by searching the web?

- https://github.com/bazelbuild/bazel/commit/f6cccae5b6f9c0ad0e7d0bf7bd31ea1263449316

**Comment by fmeum** [comment]
@beasleyr-vmw If you would like to see this fix in 5.3.1, you could cherry-pick it onto the `release-5.3.1` branch and submit a PR.

**Comment by aiuto** [comment]
Exactly what @fmeum said.  Please send a PR.
